ui/app/src/state/epics/pathNavigator.ts

```diff
@@ -284,7 +299,14 @@ export default [
               })
             );
           } else {
-            return fetchItemWithChildrenByPath(site, path, { excludes, limit, offset, keyword }).pipe(
+            return fetchItemWithChildrenByPath(site, path, {
+              excludes,
+              limit,
+              offset,
+              keyword,
+              sortStrategy: state.pathNavigator[id].sortStrategy,
+              order: state.pathNavigator[id].order
```

Comment:
Let's check why the other come from payload and these don't. It might be ok, but check.

Reply:
I added the other props into the payload for it to be consistent. But in the update config scenario it doesn't make the tree update, since `pathNavigatorFetchParentItems` is called by the `pathNavigatorInit`, that is only called when there is no state set (not on updates).